server/pulp/server/async/app.py

```diff
@@ -72,7 +166,9 @@ def initialize_worker(sender, instance, **kwargs):
     initialization.initialize()
 
     # Delete any potential old state
-    tasks._delete_worker(sender, normal_shutdown=True)
+    existing_worker = Worker.objects(name=sender).first()
+    if existing_worker:
```

Comment:
I suppose that if our current prime directive is stability, it would be better to do it in the PR (which, it's probably easier to do now than later anyway).

I will just put it back to the way it was before: I changed it because I figured it cause unnecessary and nominally inaccurate logging, but it's not a big deal and probably not worth the hassle.